tools/dartsdk/fetch.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euxo pipefail
+
+if [ ! -d "x64" ]; then
+  curl -L "https://drive.google.com/u/0/uc?id=1pJKuL6bo1OSnQMBKWT3pTGOQuMkuME8U&export=download" | tar xvf -
```

Comment:
uber-nit: Shall we create a brand new github repo (not this frb repo, surely, to avoid making git clone bloated)? Then we can, for example:

* Also put compile scripts there
* Version control it, for example, when new dart sdk releases we can upgrade a new version
* No external dependencies in addition to github

Anyway just a uber-nit :)

Reply:
We could leave this as a todo for the future, it might even be useful to other people as well.

Reply:
Sure :)

> it might even be useful to other people as well

Definitely - anyone who integrates Dart/Flutter with other languages via the C api will want this